code/Equipment/Weapons/RemoteDetonateWeapon.cs

```diff
@@ -0,0 +1,60 @@
+using Grubs.Equipment.Gadgets.Projectiles;
+using Grubs.Pawn;
+using Sandbox;
+
+namespace Grubs.Equipment.Weapons;
+
+public sealed class RemoteDetonateWeapon : Weapon
+{
+	private ExplosiveProjectile Projectile { get; set; }
+
+	bool ProjectileExploded;
```

Comment:
Access modifier needed. If it's private and NOT a property, use the `_projectileExploded` naming scheme. Otherwise, keep the name as-is.